web/src/app/modules/views/side/modules/links-actions/components/links-actions.component.ts

```diff
@@ -54,4 +56,24 @@ export class LinksActions {
     public get canExportToALM(): boolean {
         return this.additionalInformationService.canExportToALM;
     }
+
+    public toggleDescription() {
+        this.descriptionVisible = !this.descriptionVisible;
+    }
+
+    public get descriptionCollapsibleLabel(): string {
+        if (this.requirement.description.length > 30 && !this.descriptionVisible) {
+            return 'show more';

```

Comment:
Use translation service for this or show icon e.g. ```angle-double-{right|left}```. If you use the translation service, see ```TranslateService.instant(...)```.